gpii/node_modules/gpii-oauth2/gpii-oauth2-authz-server/test/js/AuthGrantFinderTests.js

```diff
@@ -38,15 +38,15 @@ https://github.com/GPII/universal/blob/master/LICENSE.txt
     });
 
     fluid.defaults("gpii.tests.oauth2.authGrantFinder.testEnvironment", {
-        gradeNames: ["gpii.tests.oauth2.pouchBackedTestEnvironment"],
-        dbViewsLocation: "../../../gpii-oauth2-datastore/dbViews/views.json",
-        dbName: "auth",
+        gradeNames: ["gpii.tests.dbOperation.pouchBackedTestEnvironment"],
+        dbViewsLocation: "../../../../../../testData/dbData/views.json",
```

Comment:
Use a module-relative reference here, and if you can't, make it so that you can :)

Reply:
This usage only happens in browser tests, I don't find the equivalent of node.js module concept with the javascript frontend. The only thing I can think of is to use [<base> tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/base) to define a base url but it doesn't help this case.